lib/python/satos_payload_sdk/app_framework.py

```diff
@@ -221,7 +222,13 @@ def stage_file_download(self, loc):
         return resp
 
     def payload_power_control(self):
-        raise NotImplementedError()
+        # In case if True-twin mode, return success
+        if api_common.g_KEEPALIVE_ENABLE == '1':
+            return api_types.AntarisReturnCode.An_SUCCESS 
```

Comment:
what about CPP application?

Reply:
It will be different PR

Reply:
I think it is better to add that in this PR itself.

Reply:
It is already handled in c++ application